controllers/license/license.js

```diff
@@ -164,6 +187,8 @@ function getCompendium(passon) {
 
             // no job for the given id available
             if(error) {
+                error.msg = 'error accessing o2r';
+                error.status = 404;
                 reject(error);
                 return;
             }
```

Comment:
(comment belongs to next line:) `JSON.parse(..)` reads from file system, which is potentially resource intensive. Do this globally once, not during every function call.